.bazeliskrc

```diff
@@ -1,2 +1,3 @@
 # See https://github.com/bazelbuild/bazelisk
-USE_BAZEL_VERSION=6.0.0
+# As per https://b.corp.google.com/issues/302171264#comment5, 6.4.0 supports Java 21. 
```

Comment:
I would use b/302171264#comment5. Avoid putting Google-internal system URL.

Reply:
thanks, removed it.